pypicloud/access/_ldap.py

```diff
@@ -0,0 +1,391 @@
+"""LDAP authentication plugin for pypicloud."""
+
+
+try:
+    import ldap  # pylint: disable=F0401
```

Comment:
Instead of the pylint disable flag, you can just put `python-ldap` in the `requirements_build.txt` file. That'll be required later anyway if any tests are written for it